tsd.json

```diff
@@ -1,6 +1,6 @@
 {
   "version": "v4",
-  "repo": "borisyankov/DefinitelyTyped",
+  "repo": "chaosmail/DefinitelyTyped",
```

Comment:
basically yes, the problem is that both angular-protractor and jquery use the global $ variable. this leads to a typescript conflict/error. I propsed a PR to DefinitelyTyped to allow a noConflict version of jquery - however it's not straight forward. therefore i forked the repo, using a noConflict version of jQuery